src/app/common/components/formfield-text.ts

```diff
@@ -41,8 +45,9 @@ import { CommonDialogsService } from '../services/common-dialogs.service';
 		</p>
 
 		<label class="visuallyhidden" [attr.for]="inputName" *ngIf="ariaLabel">{{ ariaLabel }}</label>
-		<div class="forminput-x-inputs">
+		<div class="">
 			<input
+				class="tw-h-6 md:tw-h-8 tw-w-full tw-p-2 md:tw-p-4 tw-border tw-border-solid tw-border-indigo-500 tw-rounded-[7px] md:tw-rounded-[10px] tw-bg-white"
```

Comment:
These changes also apply to all other formfields and break other pages:

![image](https://github.com/mint-o-badges/badgr-ui/assets/2675557/9aa07dae-c426-4ee7-9aee-6f4216d4fba1)


The best solution would be to create a nice input fields that works on all pages. But might take too long for now.

Reply:
Jonas already wrote that. Nevermind i am blind ;)

Reply:
I tried it. But it still looks like the styles are applied on other places too.